test/integration/test_migration_runner.py

```diff
@@ -117,12 +117,12 @@ def test_migration_table_is_created(self):
 
         assert migrtion_table_exists
 
-    def test_migration_to_0_2_0(self):
+    def test_migration_to_0_2_1(self):
         try:
             # we must use -f (force) in the case that contessa.__version__ possibly do not match our testing
             # migration version
             migration.main(
-                ["-u", SQLALCHEMY_URL, "-s", DATA_QUALITY_SCHEMA, "-v", "0.2.0", "-f"]
+                ["-u", SQLALCHEMY_URL, "-s", DATA_QUALITY_SCHEMA, "-v", "0.2.1", "-f"]
```

Comment:
Can we use fixture here? for the version to migrate to or for contessa version ? or why do you need to change this always? i thought that `-f` is fixing that.

Reply:
no we won't it change always. this is test for migration to new version, initialy this version was 0.1.5 then we decided to change it to 0.2.0 and now we had to change it to 0.2.1

in normal scenario this changes won't be common and every time for new migration we will create new set of tests